src/reducers/fileTree.tsx

```diff
@@ -372,6 +376,75 @@ export const goToRelativeFile = ({
   };
 };
 
+type GoToRelativeMessageParams = {
+  _getRelativeMessage?: typeof getRelativeMessage;
+  _viewVersionFile?: typeof viewVersionFile;
+  currentMessageUid: LinterMessage['uid'] | void;
+  currentPath: string;
+  messageMap: LinterMessageMap;
+  pathList: string[];
+  position: RelativePathPosition;
+  versionId: number;
+};
+
+export const goToRelativeMessage = ({
+  /* istanbul ignore next */
+  _getRelativeMessage = getRelativeMessage,
+  /* istanbul ignore next */
+  _viewVersionFile = viewVersionFile,
+  currentMessageUid,
+  currentPath,
+  messageMap,
+  pathList,
+  position,
+  versionId,
+}: GoToRelativeMessageParams): ThunkActionCreator => {
+  return async (dispatch, getState) => {
+    const nextRelativeMessageInfo = _getRelativeMessage({
+      currentMessageUid,
+      currentPath,
+      messageMap,
+      pathList,
+      position,
+    });
+
+    if (nextRelativeMessageInfo) {
+      const { line, path, uid } = nextRelativeMessageInfo;
+      const {
```

Comment:
This could fit on one line

Reply:
For some reason each time I put that on a single line, `prettier` changes it back to look like this. Not sure why. Do you want to give it a try and see if the same happens to you?

Reply:
ah, ok, then no worries. Prettier is the boss.